doc/dsdr/0005-leaver-transition.md

```diff
@@ -0,0 +1,19 @@
+# Leaver
+
+## Context
+
+We need an agreed upon definition of what a "leaver" is. 
+
+## Decision
+
+A leaver is someone whose EHCP has ended or has moved out of the Local Authority and therefore the Local Authority are no longer responsible for their education and support. A leaver for 2016 would be someone who between Jan-16 and Jan-17 transitioned out of SEND.
```

Comment:
or has moved beyond the maximum year of the SEND system?

Reply:
So I did have this but then removed it, as I think that for someone aging out they would qualify as "someone whose EHCP has ended". I purposefully didn't expand on why someone's EHCP might end, as LAs have different opinions on how it is/should be possible or not. Thoughts?

Reply:
"whose EHCP has ended due to age or other reason"